Tasks/ArchiveFiles/archivefiles.ts

```diff
@@ -1,7 +1,7 @@
 import path = require('path');
 import tl = require('vsts-task-lib/task');
 
-var repoRoot: string = tl.getVariable('build.sourcesDirectory');
+var repoRoot: string = tl.getVariable('System.DefaultWorkingDirectory');
```

Comment:
I don't think System.DefaultWorkingDirectory is defined in Build. In many tasks we have been using tl.getVariable('build.sourcesDirectory') || tl.getVariable('System.DefaultWorkingDirectory'); 

It will be good to cross check this.